### PR TITLE
Add missing directory stem prefix to some #include directives

### PR DESCRIPTION
**Pull request by 0xdaryl**
Signed-off-by: Daryl Maier <maier@ca.ibm.com>

### CODE REVIEW

**Comment by 0xdaryl** [comment]
genie-omr build all

**Comment by 0xdaryl** [comment]
The zOS failure is not related to these compiler changes.

**Comment by 0xdaryl** [comment]
genie-omr build all

File: compiler/runtime/CodeCacheTypes.cpp [hunk 1]
```diff
@@ -19,7 +19,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
  *******************************************************************************/
 
-#include "CodeCacheTypes.hpp"
+#include "runtime/CodeCacheTypes.hpp"
```
Comment (fjeremic) [inline compiler/runtime/CodeCacheTypes.cpp]: We don't have to do this here as we are inconsistent elsewhere but what do you think about alphabetizing the include directives in the compiler subdirectories?

**Comment by fjeremic** [comment]
The tags seem to imply this is a bug. Does it close a particular issue? And if not a minor comment in the commit on why this is a bug per-say should be warranted.

File: compiler/runtime/CodeCacheTypes.cpp (hunk 1 above)
Comment (mstoodle) [reply compiler/runtime/CodeCacheTypes.cpp]: not an unreasonable goal to move towards, I think

**Pull request merged**